trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesTermsService.php

```diff
@@ -177,15 +192,17 @@ public function loadTerms($schema = NULL) {
    *   method: Collection Method.
    *   name: Name/Germplasm line.
    *   experiment_container: Plot.
-   *   unit_to_method_relationship_type: Related - create relationships (unit - method).
-   *   method_to_trait_relationship_type: Related - create relationships (method - trait).
+   *   unit_to_method_relationship_type: Related - create relationships
+   *   (unit - method).
+   *   method_to_trait_relationship_type: Related - create relationships
+   *   (method - trait).
    *   experiment_replicate: Planting replicate.
    *   unit: Unit of measurement.
    *   experiment_year: Year.
```

Comment:
Github won't allow suggestions higher than line 192, but I suggest adding bullets for data_collector, entry, genus and location as well.

```suggestion
   *   - method: Collection Method.
   *   - name: Name/Germplasm line.
   *   - experiment_container: Plot.
   *   - unit_to_method_relationship_type: Relate unit to method.
   *   - method_to_trait_relationship_type: Relate method to trait.
   *   - experiment_replicate: Planting replicate.
   *   - unit: Unit of measurement.
   *   - experiment_year: Year.
```

Reply:
I found out that this list is outdated, please see my comments https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/138#issuecomment-2588424283

Reply:
Fixed the outdated list and applied the bullets in commit - Updates term service outdated doc block config name references and applied bullets